packages/db/test/cli/gen-types.test.mjs

```diff
@@ -9,21 +9,31 @@ import { urlDirname } from '../../lib/utils.js'
 
 t.jobs = 1
 
+let counter = 0
+
 const pathToTSD = path.join(urlDirname(import.meta.url), '../../node_modules/.bin/tsd')
 
 t.test('generate ts types', async (t) => {
   const testDir = path.join(urlDirname(import.meta.url), '..', 'fixtures', 'gen-types')
-  const cwd = path.join(urlDirname(import.meta.url), '..', 'tmp', 'gen-types-clone-1')
+  const cwd = path.join(urlDirname(import.meta.url), '..', 'tmp', `gen-types-clone-${counter++}`)
+
+  try {
+    await rm(cwd, { recursive: true })
```

Comment:
You may want to specify retries here for Windows. Same comment for the other uses of `rm()` in this PR.